### PR TITLE
lsd 0.11.1 (new formula)

### PR DESCRIPTION
**Pull request by colindean**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/lsd.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  url "https://github.com/Peltoche/lsd/archive/0.11.1.tar.gz"
+  sha256 "7c9cdde049e02433a94cbfc3e7efd2b2ff7b08b9e5343849069b896ea646e40e"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix, "--path", "."
+  end
+
+  def caveats; <<~EOS
```
Comment (fxcoudert) [inline Formula/lsd.rb]: We reserve `caveats` for Homebrew-specific information. This is generic documentation, not specific to Homebrew, so the `caveats` should be removed.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
@colindean I went ahead and merged, I hope you don't mind

**Comment by colindean** [comment]
Not at all! I don't think I have commit bit on this repo.

**Comment by fxcoudert** [comment]
Oh, I thought the “member” badge was linked to commit rights, but it's not. Sorry.

**Pull request closed without merging**